### DISCUSSION TITLE
Can all three folder names be renamed without issues

### DISCUSSION THREAD

**Discussion by ebitogu** [serverpod/serverpod#2032] [Q&A] [answered]
Thanks for this amazing framework.

However, I would love to change all three directory names in my project for improved command line navigation.

I just wanted to know if the name changes will break any thing in the process.

**Rationale:**
- The current directory names (`tomatoe_flutter`, `tomatoe_client`, `tomatoe_server`) are verbose and require longer commands (`cd tomatoe_client`) for navigation.
- Since they already share same directory name during the creation process renaming them to shorter names (`app`, `client`, `server`) would significantly improve command line navigation efficiency.
- Shorter directory names are easier to remember and type

Thank you

**Comment by SandPod** [comment, marked as answer]
Hey @ebitogu!

Yes, the names of the folders can be changed.
- The `_flutter` folder can be changed arbitrarily since nothing depends on that folder.
- The `_ server` folder can be change arbitrarily.
- Changing the name of the `_client` folder requires the following changes:
  1. Update the `client_package_path` in the generator configuration located in `_server/config/generator.yaml` to your new folder name. This ensures that we can generate new client code. 
  2. Update the name of the package in the pubspec to match the new folder name, this ensures that the import resolve correctly in the `_flutter` package. (Optional step here is to also rename the exported library file to match the new package name located in `*_client/lib/*_client.dart`.
  3. Update the import path to the `_client` in the `_flutter` package using the new folder and package name.
  4. Update the import in flutter code using the new client package name.


Example of step two can be seen here: https://github.com/serverpod/serverpod/blob/db5df4b21b06bf9ef34af5fcaf216906027ec307/packages/serverpod/config/generator.yaml#L3

**Comment by ebitogu** [comment]
Okay, that was really helpful. Thank you
